File: src/main/java/amerifrance/guideapi/objects/Category.scala
package amerifrance.guideapi.objects

import java.util

class Category(entries: util.List[Entry] = new util.ArrayList[Entry](), unlocCategoryName: String) {

  def addEntry(entry: Entry) = {
    this.entries.add(entry)
  }

  def addEntryList(list: util.List[Entry]) = {
    this.entries.addAll(list)
  }

  def removeEntry(entry: Entry) = {
    this.entries.remove(entry)
  }

  def removeCategoryList(list: util.List[Entry]) = {
    this.entries.remove(list)
  }
}

File: src/main/java/amerifrance/guideapi/objects/Category.java
package amerifrance.guideapi.objects;

import java.util.ArrayList;
import java.util.List;

public class Category {

    public List<Entry> entries;

    public Category() {
        this.entries = new ArrayList<Entry>();
    }

    public Category(List<Entry> entryList) {
        this.entries = entryList;
    }
}
